service/history/replication/cache.go

```diff
@@ -145,6 +169,6 @@ func (h *int64Heap) Pop() interface{} {
 	return x
 }
 
-func (h *int64Heap) Peek() int64 {
```

Comment:
Is this used anywhere? If you're changing the method signature and if the cache is pretty general, this could be a generic type?